htmd/parameterization/dihedral.py

```diff
@@ -154,23 +117,19 @@ def _setup(self):
             raise ValueError('The number of dihedral and QM result sets has to be the same!')
 
         # Get dihedral names
-        self._names = ['-'.join(self.molecule.name[list(dihedral)]) for dihedral in self.dihedrals]
+        self._names = ['-'.join(self.molecule.name[dihedral]) for dihedral in self.dihedrals]
 
         # Get equivalent dihedral atom indices
         self._equivalent_indices = []
-        for dihedral, name in zip(self.dihedrals, self._names):
-            for equivlantet_diherals in self.molecule._parameterizable_dihedrals:
-                if equivlantet_diherals[0] == dihedral:
-                    self._equivalent_indices.append(equivlantet_diherals)
+        for idihed, dihedral in enumerate(self.dihedrals):
+            found = False
+            for rotatableDihedral in self._rotatable_dihedrals:
```

Comment:
`rotatable`--> `parameterizable`. I'm trying to make this consistent thought the code.

Reply:
renamed